frontend/src/js/controllers/featuredChallengeCtrl.js

```diff
@@ -30,19 +30,9 @@
         vm.loginContainer = angular.element('.exist-team-card');
```

Comment:
@aayusharora make this variable a global container name like loaderContainer instead of loginContainer and do changes accordingly.